src/components/Accordion.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { render } from "src/utils/rtl";
+import { Accordion } from "./Accordion";
+
+describe(Accordion, () => {
+  it("displays the correct content", async () => {
+    // Given an accordion component
+    // When rendered
+    const r = await render(<Accordion title="Test title"> Test description </Accordion>);
+
+    // Then it shows the correct title
+    expect(r.accordion_title()).toHaveTextContent("Test title");
+    // And the description will be hidden by default
+    expect(r.accordion_details()).toHaveStyleRule("max-height", "0");
+    // And the border bottom is not displayed by default
+    expect(r.accordion_container()).not.toHaveStyleRule("border-bottom-width", "1px");
+    // And the dropdown button is enabled by default
+    expect(r.accordion_title()).not.toHaveAttribute("disabled");
+  });
+
+  it("can expand by default", async () => {
+    // Given an accordion component with defaultExpanded set
+    // When rendered
+    const r = await render(
+      <Accordion title="Test title" defaultExpanded>
+        {" "}
+        Test description{" "}
```

Comment:
**nit (non-blocking):** I'm guessing the formatter did this, but those `{" "}` are unnecessary for the test.